srv/WriteVarPosition.srv

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2024, Yaskawa Europe, GmbH
+# SPDX-FileCopyrightText: 2024, Delft University of Technology
+#
+# SPDX-License-Identifier: Apache-2.0
+
+uint32 var_number
+
+# only poses defined in 'base' frame will be accepted
+#
+# NOTE: the ROS-Industrial 'base' frame, not Yaskawa Base frame
+#
+# NOTE 2: it's the responsibility of the client to transform poses into this
+#         frame. The server will reject poses not already in 'base' frame.
+geometry_msgs/PoseStamped value
```

Comment:
I would suggest using `geometry_msgs/Transform` instead of `PoseStamped`. That is what's already used in PositionMonitor.c for the `/tf` generation.

Reply:
> it's the responsibility of the client to transform poses into this frame.

What is the motivation here? Since a robot P variable can be different formats, do we need to force the user to use cartesian? 

Perhaps this could be a 'union' of a `Transform` and a `sensor_msgs/JointState`. We could populate one (or both) of them based on the type of the P var?

Reply:
This is my initial version (from way back when: https://github.com/Yaskawa-Global/motoros2/issues/180#issuecomment-1770539252).

@ted-miller: is a P var a pose/position, or a transformation. Semantically these are different.

re: both a joint and Cartesian pose: yes, we could do that. It wouldn't be a union strictly speaking, but we can define semantics where an uninitialised `JointState` field would need the Cartesian pose to be valid and vice versa.

In any case: it would be best to require Cartesian poses to be in the robot's `base` frame already (or whichever frame makes the most sense for P variables) as transforming on-robot is much more difficult.